Makefile

```diff
@@ -16,12 +16,12 @@ GO_ARGS=-tags '$(GO_TAGS)'
 
 # Test vars can be used by all recursive Makefiles
 export GOOS=$(shell go env GOOS)
-export GO_BUILD=env GO111MODULE=on go build $(GO_ARGS)
-export GO_TEST=env GO111MODULE=on go test $(GO_ARGS)
+export GO_BUILD=env FLUX_PARSER_TYPE=rust GO111MODULE=on go build $(GO_ARGS)
```

Comment:
Question:

Why is this set at compile-time?

Reply:
For `go build` I think you're right, it won't have any affect.  Some of the `go` commands run the built executable (like `go test`) so I wanted to make sure that I preserve the existing behavior of using the Rust parser whenever we have a libflux build.

One thing I didn't do here was set the environment variable for the `generate` step, since it does not use the libflux build either.  So currently we still use the Go parser to generate the `_gen.go` files we use to export the standard library (just like before this PR merged).  But we could probably start using the Rust parser there.